EastAgile/twitterbot
# [f] 147 - twitterbot read options from a configuration file

## Description

For story [147 - twitterbot read options from a configuration file](https://www.pivotaltracker.com/story/show/117879147)
* Developer can create a `.twitterbot` file to config the bot
* Configuration from cli arguments and config file are merged

## How to test
* Run `$ start_twitterbot` it should throw an exception
* Create a `.twitterbot` file in the working directory, now the bot should not raise any error

```
[handlers]
package = handlers.module

[auth]
consumer_key = 123
consumer_secret = 123
token_key = 123
token_secret = 123
```
